Ready for review

BrittanyBlake opened the pull request:
## Positioning and Floating Elements: New York Times Clone
-Added a fixed navigation bar
-Added article content
-Added embedded images and videos
-Added aside bar
-Added a footer

[Live Version](https://brittanyblake.github.io/NewYorkTimes-clone/)

BrittanyBlake committed:
added navbar

BrittanyBlake committed:
add CSS to header

Morrism1 committed:
Add images folder

Morrism1 committed:
Add html body content

Morrism1 committed:
Add the styling to the body title and image

Morrism1 committed:
Add more elements in index.html

Morrism1 committed:
Add styles to paragraphs and the aside bar

Morrism1 committed:
Merge pull request #1 from Morrism1/feature

Feature

Morrism1 committed:
Modify image in image folder

Morrism1 committed:
Finish adding the rest of the body elements

Morrism1 committed:
Finish styling the rest of the paragraphs and positioning elements

Morrism1 committed:
Merge pull request #3 from Morrism1/feature

Feature

BrittanyBlake committed:
add bottom section and form element

BrittanyBlake committed:
style bottom section.

BrittanyBlake committed:
Add section with more articles

BrittanyBlake committed:
Style section with more articles

BrittanyBlake committed:
Add most popular and editors' picks sections

BrittanyBlake committed:
Style most popular and editors' picks

Morrism1 committed:
Merge pull request #1 from BrittanyBlake/feature

Feature

Morrism1 committed:
Add footer content

Morrism1 committed:
Style footer content

Morrism1 committed:
Merge pull request #6 from Morrism1/feature

Feature

BrittanyBlake committed:
Add stickler file

BrittanyBlake committed:
Add stylelint file

BrittanyBlake committed:
update stickler and css

BrittanyBlake committed:
fix linter errors

BrittanyBlake committed:
Add README

BrittanyBlake committed:
add screenshot of webpage

BrittanyBlake committed:
Fix linter errors

fatymahmed reviewed (commented):
## Hi @BrittanyBlake @Morrism1, 
Great job guys,👏👏 very nice description for your pull request 😀, there are just a few things that you need to change for it to be better and approved.

**Navigation bar**
- It has a "Hamburger **button**" on the leftmost side, kindly change it to a button.
- The NY Times logo should be a link.

**Main Content**
- Make sure your text is aligned with the image.
This is how it looks on the new york page.
![screenshot](https://user-images.githubusercontent.com/16196289/74013448-851a2e00-499d-11ea-974e-288f4aefc6b9.png)
This is how it looks on yours.
![screenshot](https://user-images.githubusercontent.com/16196289/74013456-88151e80-499d-11ea-8c6b-d04ce652e314.png)

**Bottom part**
- For the _More in Space and Astronomy_ section, the images and texts below them are **links** to other articles.

**Footer**
- It should have a fixed position (it sticks to the bottom part of the screen) -  This is for the _Access more of the Times by creating a free account or logging in_ part.

**Implementation details**
**The Theory of Inflation**
- I noticed that you are using an image with the text included in the image for this part, however, you should use the image without the text (you can get it from the new york times page)then add the text by yourself.

**General**
- Check HTML markup with this validator https://validator.w3.org/#validate_by_upload as [mentioned in Pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/validation_requirement.png), you have some errors, fix them.
![Screenshot](https://user-images.githubusercontent.com/16196289/74014921-b47e6a00-49a0-11ea-9c68-fc03a43e5344.png)
- You could add some space between the main content and the aside section, this is how it looks.
![screenshot](https://user-images.githubusercontent.com/16196289/74015566-ee03a500-49a1-11ea-805c-3b4f36240654.png)

**[OPTIONAL]**
- You are encouraged to use the `<main>` tag for the dominant content of the `<body>` of a document.
-  For the use of absolute positioning for the editor's picks and subscribe sections, you can easily achieve the expected outcome with the use of the `<section>` and `<aside>` tags which help keep the document flow.

fatymahmed reviewed (changes requested):
### Kindly make the suggested changes then request for another code review.
### Status: Changes Required
_Feel free to reach out to me on slack (Fatima) if you have any questions, I'd be happy to help._

fatymahmed commented on style.css:
```diff
@@ -0,0 +1,696 @@
+  width: 59em;
+}
+
+.main .caption {
+  text-align: left;
+  margin-left: -170px;
```
- You can make this be aligned with the image by changing the `margin-left` .

BrittanyBlake committed:
fix required changes

BrittanyBlake committed:
fix required changes

Morrism1 committed:
Merge pull request #2 from BrittanyBlake/gh-pages

Get update

BrittanyBlake committed:
Add fixed footer

BrittanyBlake committed:
Make footer fixed

BrittanyBlake committed:
update theory of inflation image

BrittanyBlake committed:
fix linter errors

Morrism1 committed:
Merge pull request #3 from BrittanyBlake/gh-pages

New Changes

Morrism1 committed:
Add text paragraph on the inflation image

Morrism1 committed:
Style and positioning of text paragraph on the inflation image

Morrism1 committed:
Merge pull request #11 from Morrism1/gh-pages

New changes

BrittanyBlake committed:
fix linter errors

BrittanyBlake committed:
Fix 2 html validator errors

BrittanyBlake commented:
## Positioning and Floating Elements: New York Times Clone
-Fixed navigation bar hamburger button
-Made NY Times logo a link
-Added links to images and text in "More in Space and Astronomy" section
-Added footer with fixed position
-Fixed HTML validator errors
-Added blank theory of inflation image
-Added text to the theory of inflation image


[Live Version](https://brittanyblake.github.io/NewYorkTimes-clone/)

nikasulo reviewed (approved):
# Really amazing work! @BrittanyBlake and @Morrism1 :handshake: 
This is really beautiful and I loved reviewing your project. Amazing job on the PR description as well, it was nice to read. Please merge your branch and keep moving forward. See you at the top! :rocket: 

##### Think I missed something? Need some help with this or any of your projects? Please feel free to reach out to me on [Twitter](https://twitter.com/oluwadamilareo_) or [Instagram](https://instagram.com/oluwadamilare_olusakin), even if you just want to chat and I will respond to you as soon as I can.
